about.md

```diff
@@ -5,15 +5,19 @@ permalink: /about/
 ---
 
 
-This is the technical blog of [Allo-Media], where we share our experiences and discoveries with code and the technologies we use.
+This is the technical blog of [Allo-Media], where we share our experiences and
+discoveries with code and the technologies we use.
 
-This blog is powered by [Jekyll] and hosted on [Github]; so if you spot a typo, feel free to send us a PR!
+This blog is powered by [Jekyll] and hosted on [Github]; so if you spot a typo,
+feel free to send us a PR!
 
 <div style="text-align:center;margin: 2em 0">
   <img style="width: 400px" src="https://www.allo-media.net/assets/img/vitrine/logo.png">
```

Comment:
@BeardedBear Could you please give a link to an image we could use here?

Reply:
Why a logo in this page ?

Reply:
You're right, I'm not sure it's even needed.
What do we do with this, should we remove it?

Reply:
Yes i think :)